OsmAnd/res/layout/bottom_sheet_item_with_switch_blocked_for_free.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
bottom_sheet_item_with_switch_and_button?

Reply:
they are not the same. we have compound button with icon